app/liquid_tags/golang_tag.rb

```diff
@@ -0,0 +1,38 @@
+class GolangTag < LiquidTagBase
+  PARTIAL = "liquids/golang".freeze
+  URL_REGEXP = %r{(http|https)://play.golang.com/p/[a-zA-Z0-9\-/]*}.freeze
```

Comment:
1. The example given in the PR description uses `golang.org`, the regex here `golang.com`. I guess we should support both?
2. `(http|https)` is more commonly written as `https?`
3. The final `*` should probably be a `+`, otherwise https://play.golang.org/p/ would match too.
4. We should probably anchor the regex.

```suggestion
  URL_REGEXP = %r{\Ahttps?://play.golang.(com|org)/p/[a-zA-Z0-9\-/]+\z}.freeze
```

Reply:
We actually need to only support `org` ... I will make this change